packages/app/src/components/VisualizationOptions/NoticeBox.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { NoticeBox as UiNoticeBox } from '@dhis2/ui'
+
+import { tabSectionOption } from './styles/VisualizationOptions.style.js'
+
+const NoticeBox = ({ title, text, type }) => {
+    const props = {
+        title,
+    }
+
+    if (type) {
+        props[type] = true
+    }
+
+    return (
+        <div className={tabSectionOption.className}>
+            <UiNoticeBox {...props}>{text}</UiNoticeBox>
```

Comment:
A bit magical to pass the props like this, why not pass them explicitly?

Reply:
Refactored to use the same interface as `ui` NoticeBox.